src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -47,6 +48,24 @@ public function __construct(string $kernelEnvironment, string $projectDirectory)
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('filter', InputArgument::OPTIONAL, 'The name of an environment variable or a filter.', null, $this->getAvailableVars(...)),
+            ])
+            ->setHelp(<<<'EOT'
+The <info>%command.full_name%</info> command displays all the environment variables configured by dotenv:
+
+  <info>php %command.full_name%</info>
+
+To get specific variable, specify its name:
```

Comment:
So, it can be the env name or part of the name, right?

Reply:
Updated with clarification: `To get specific variable, specify its full or partial name:`